config/routes.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
+  get 'reservations/create'
+  get 'reservations/destroy'
```

Comment:
[Optional] I think there is no point of having a get method for create and destroy. It should be defined in resources which you already did.

Reply:
I believe those lines were auto-generated. I'll proceed to remove them.